src/Core/ExceptionWrapper.php

```diff
@@ -35,12 +34,41 @@ class ExceptionWrapper
      * Adds response body to the exception message.
      *
      * @param BadResponseException $ex
-     * @return GraphException containing HTTP response from Graph API
-     * 
+     * @return BadResponseException containing HTTP response from Graph API
      */
     public static function wrapGuzzleBadResponseException(BadResponseException $ex)
     {
-        $errMsg = "Received {$ex->getResponse()->getStatusCode()} for call to {$ex->getRequest()->getUri()}\nAPI response: {$ex->getResponse()->getBody()->getContents()}";
-        return new GraphException($errMsg);
+        $response = $ex->getResponse();
+
+        // Safety check for Guzzle < 7.0
+        if (!$response) {
+            return $ex;
+        }
+
+        /** @see \GuzzleHttp\Exception\RequestException::create() */
+        if (preg_match('/^(.+: `.+ .+` resulted in a `.+ .+` response)($|:\n)/U', $ex->getMessage(), $match)) {
```

Comment:
This is to modify only potentially truncated messages, not manually-constructed exceptions like e.g. https://github.com/guzzle/guzzle/blob/7.3.0/src/RedirectMiddleware.php#L211 that purposedly *doesn't* include the response body